sdk/core/azure-core-amqp/test/ut/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+
+cmake_minimum_required (VERSION 3.13)
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+project(azure-core-amqp-tests LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+include(AzureBuildTargetForCI)
+include(GoogleTest)
+
+# Unit Tests
+add_executable(azure-core-amqp-tests
+    "amqp_value_tests.cpp"
+    "amqp_header_tests.cpp"
+    "amqp_properties_tests.cpp"
+    "amqp_message_tests.cpp"
+    "connection_tests.cpp"
+    "transport_tests.cpp" 
+    "session_tests.cpp"
+    "link_tests.cpp" 
+    "message_sender_receiver.cpp"
+    "connection_string_tests.cpp"
+ claim_based_security_tests.cpp "message_source_target.cpp" "management_tests.cpp" "azure_core_amqp_tests.cpp")
```

Comment:
inconsistent formatting and use of surrounding quotes " vs not (for example `claim_based_security_tests.cpp`).

In other places within the repo, we don't surround with quotes.